### DISCUSSION TITLE
Minor typo

### DISCUSSION THREAD

**Discussion by artgoldberg** [uwrit/leaf#527] [General]
FYI, in https://leafdocs.rit.uw.edu/installation/installation_steps/2_app_server/, the link in 

> Refer to Microsoft's current instructions for installing .NET Core framework. https://dotnet.microsoft.com/download/linux-package-manager/rhel/sdk-current.

is dead.
Arthur

**Comment by ndobb** [comment]
Thanks Arthur, this has now been corrected.
